src/arrayMethodReduce.js

```diff
@@ -5,7 +5,14 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    const start = (arguments.length === 1) ? 1 : 0;
```

Comment:
What does it mean `start` ? Your variable names should be meaningful. Better rename to `startIndex`.

Reply:
Agree))